### DISCUSSION TITLE
JWT support

### DISCUSSION THREAD

**Discussion by tylermontney** [scottoffen/grapevine#128] [Q&A]
Based on https://scottoffen.github.io/grapevine/docs/tutorials/using-jwt, Grapevine 5 _will_ or _does_ support JWT? I couldn't find any reference to JWT in the project.

**Comment by scottoffen** [comment]
It neither _will_ nor _does_, but rather _can_. That's a placeholder page for a tutorial that I haven't written yet. The skinny is that you create a route that returns a JWT, and then add middleware that validates the JWT prior to route access.

There is a brief discussion [over here](https://github.com/scottoffen/grapevine/discussions/96) and another that talks about [using local over here](https://github.com/scottoffen/grapevine/discussions/117).
